apis/python/notebooks/tutorial_soma_objects.ipynb

```diff
@@ -1380,7 +1380,328 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "836a3a97-901e-4428-8ee3-d93a49a1ee26",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "As an example, we will load a version of pbmc3k that has been generated using a `SparseNDArray`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "71f8fed4-4ffd-4f30-a5b0-4e3a4a3730f3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<SparseNDArray 'file:///opt/TileDB-SOMA/apis/python/notebooks/data/sparse/pbmc3k/ms/RNA/X/data' (open for 'r')>"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "experiment = tiledbsoma.open(\"data/sparse/pbmc3k\")\n",
+    "X = experiment.ms[\"RNA\"].X[\"data\"]\n",
+    "X"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76935223-eda5-48c0-89f5-26e5bfdf3628",
+   "metadata": {},
+   "source": [
+    "Let's take a look at the schema:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "41897a5c-2225-49f9-b9f2-3a68a6ad8079",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "soma_dim_0: int64\n",
+       "soma_dim_1: int64\n",
+       "soma_data: float"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "X.schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ff9fcfd3-b456-430c-9a98-6cd46d2dd9d2",
+   "metadata": {},
+   "source": [
+    "This is the same as the `DenseNDArray` version, which makes sense since it's still a 2-dimensional matrix with `float` data.\n",
+    "\n",
+    "Let's look at the shape:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "id": "42c1d852-6492-4a5e-b1fe-bc9af3f83639",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(9223372036854773760, 9223372036854773760)"
+      ]
+     },
+     "execution_count": 43,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "X.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f91e73f-aa79-4fa2-a763-ad792e5641a8",
+   "metadata": {},
+   "source": [
+    "Since sparse matrices are not represented as contiguous arrays in memory, they don't have a fixed _size_ (or shape) like a dense matrix would have. Instead, `.shape()` returns the _capacity_ of the matrix, which means that those are valid indices for reading/writing to that matrix. These are dependent on the capacity of the system rather than the actual size of the array.\n",
```

Comment:
```suggestion
    "Since sparse matrices are not represented as contiguous arrays in memory, they don't have a fixed size like a dense matrix would have. Instead, `.shape()` returns the _capacity_ of the matrix, which means that those are valid indices for reading/writing to that matrix. These are dependent on the capacity of the system rather than the current bounding box of the array.\n",
```